### DISCUSSION TITLE
Why is there a flag of Catalonia in the registry of donors to Linux Mint?

### DISCUSSION THREAD

**Discussion by adrianlopezgalera** [linuxmint/discussions#70] [Questions] [answered]
Hello everybody,

Hours ago I posted this question on the Community Discord server. **I have been unilaterally kicked out by a moderator for the simple fact of raising a political issue and defending myself against his personal attacks**. I haven't even been able to get around to reading other members' responses.

I have been unfairly accused of wanting to promote some kind of political activism or signaling to the community of any political affiliation. I think there has been a complete lack of interest and immaturity in dealing with this issue by mere social media volunteers with no knowledge of law.

Therefore, since I would like an answer from those responsible of Linux Mint, I direct the same question here, as well as my exposed and ignored arguments on the Discord server:

[Discussion in which I was banned without justification](https://discord.com/channels/628978428019736619/1127884253644984320)

If we look at the registry of donors to Mint Mint, we see, for example, that in the last month published on the blog there was a donor who appears with the Catalonia flag.

Catalonia is a region of Spain. It is not an independent country recognized inside or outside of Spain.

I do not intend to start a political discussion, it just seems to me an obvious mistake that a regional flag is between national flags. If this possibility is offered for Catalonia, then the regional flags of other countries should also appear, including those of the other regions of Spain. No?

That's inconsistent, unless there is also the possibility of adding non-countries and fantasy locations. It reduces the credibility of an international project that allows this because, from the Spanish point of view, it is endorsing the independence ideology of a region.

 **I'm sorry if my question bothers or offends**. I'm not a computer scientist, just a user who has supported the community through donations, suggestions and bug finding. It saddens me very much that these issues are trivialized or that they are interpreted as "political activism" or "desire to annoy." 

I am a person quite grown in years, I am not a young adult who spends his afternoons on the Linux Mint server and Minecraft at night. I don't understand how there can be so many users with so little reading and understanding ability. I am not here (ot there) for fun, but to contribute in the way that I can as a mature and rational person.

It seems unfair to me and I reproach that I have been expelled on Discord. As often happens, the middle managers of a company or organization often abuse their power and rise through the ranks until they reach the peak of their incompetence.

Regards.

_______________________

![Captura de pantalla de 2023-07-10 10-20-20-(2)](https://github.com/linuxmint/discussions/assets/77041822/eb1c30d2-7976-4e58-8dba-8cc0458a6e51)

**Comment by JosephMcc** [comment]
I'm locking this for now simply because no one needs to comment on it outside of the people who run the project. This really isn't the place for political discussions and I don't want it to devolve into something it shouldn't.

**Comment by clefebvre** [comment, marked as answer]
Hi @Julvenzor,

We give donors the ability to set their location. Note that these flags indicate a location, not a "country". Whether that location is indeed a country is:

- political
- subjective
- none of our business

We don't have all possible locations in our database, only the ones we received donations or mirrors from. When a donor requests the addition of a location, or a mirror comes from somewhere we don't have, we add it to the database.

Note that https://linuxmint.com/mirrors.php also lists locations, not countries.
